Vienna/Sources/Models/FeedItem.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  FeedItem.swift
+//  Vienna
+//
+//  Copyright 2019
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  https://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public class FeedItem: NSObject {
+
+    @objc var title: String
```

Comment:
You should use default values here instead of overriding the initialiser. It looks cleaner then. Example:
```swift
@objc var feedItemDescription = ""
```